### ISSUE TITLE
AdminClient AlterOperation DELETE

### ISSUE THREAD

**Issue by speedfl** [confluentinc/confluent-kafka-go#883]
Description
===========

In java sdk, admin client offers `org.apache.kafka.clients.admin.AlterConfigOp.OpType.DELETE` 

Admin client only offers `SET`

How to reproduce
================

https://github.com/confluentinc/confluent-kafka-go/blob/master/kafka/adminapi.go#L230

Checklist
=========
Please provide the following information:

 - [X] confluent-kafka-go and librdkafka version (`LibraryVersion()`): v1.9.2
 - [X] Apache Kafka broker version: N/A
 - [X] Client configuration: `ConfigMap{...}`: N/A
 - [X] Operating system: N/A
 - [X] Provide client logs (with `"debug": ".."` as necessary): N/A
 - [X] Provide broker log excerpts: N/A
 - [X] Critical issue: N/A

**Comment by mhowlett** [comment]
yep, it looks like the additional operations aren't implemented. i see that they are in librdkafka - it's just a bit of work to add them to the binding.
